src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,17 @@
+body,
+div,
+p,
+a {
+  margin: 0;
+  font-family: $font-family;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+p {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
```

Comment:
You should not set such styles to tag name selector. Those are not 'reset' styles